vhdl_lang/src/ast.rs

```diff
@@ -667,6 +674,7 @@ pub struct ProcedureSpecification {
 }
 
 /// LRM 4.2 Subprogram declaration
+#[with_token_span]
```

Comment:
For my purposes it is enough if `SubprogramDeclaration` has the `TokenSpan` attached to it. If you wanted to get the tokens for only the specification part of a `SubprogramBody`, you would need it, though.

But since that is not needed right now, I will remove it from `SubprogramSpecification`.